app/mesh/konnect-global-control-plane.md

```diff
@@ -4,17 +4,16 @@ content_type: reference
 description: 'Set up a global Control Plane in {{site.konnect_short_name}}, add a zone, and deploy the Kubernetes demo app to test your {{site.mesh_product_name}} mesh.'
 layout: reference
 breadcrumbs:
-  - /mesh-manager/
+  - /mesh/
 products:
   - mesh
 works_on:
   - konnect
 tags:
-  - mesh-manager
   - service-mesh
 ---
 
-Using Mesh Manager, you can create global Control Planes to manage your {{site.mesh_product_name}} meshes. This guide explains how to configure a global Control Plane and then install the Kubernetes demo app to test out the {{site.mesh_product_name}} interface in {{site.konnect_short_name}}.
+Using the Mesh GUI in {{site.konnect_short_name}}, you can create global Control Planes to manage your {{site.mesh_product_name}} meshes. This guide explains how to configure a global Control Plane and then install the Kubernetes demo app to test out the {{site.mesh_product_name}} interface in {{site.konnect_short_name}}.
```

Comment:
```suggestion
Using the Mesh GUI in {{site.konnect_short_name}}, you can create global control planes to manage your {{site.mesh_product_name}} meshes. This guide explains how to configure a global Control Plane and then install the Kubernetes demo app to test out the {{site.mesh_product_name}} interface in {{site.konnect_short_name}}.
```

Kong Mesh meshes

feels weird here but I don't know if there is another way.

Reply:
Changing to "instances"